app/components/Zoom/Zoom.js

```diff
@@ -3,17 +3,20 @@ import { css } from 'emotion';
 import { useSelector } from 'react-redux';
 import { LangContext } from 'containers/Service/useLangReducer';
 import useWindowSize from 'hooks/useWindowSize';
+import setZoomState from 'state/setZoomState';
+import { useRecoilValue } from 'recoil';
 
 export const ZoomContext = createContext(null);
 
 const Zoom = ({ children, isBibleParallel = false }) => {
     const [width] = useWindowSize();
     const { lang } = useContext(LangContext) || {};
     const isNestedZoom = useContext(ZoomContext);
-    const zoom = isNestedZoom
-        ? 1
-        : ((lang === 'parallel' || isBibleParallel) && width < 480 ? 0.8 : 1) *
-          useSelector(state => state.settings.zoom);
+    const zoom = useRecoilValue(setZoomState);
+    // const zoom = isNestedZoom
+    //     ? 1
+    //     : ((lang === 'parallel' || isBibleParallel) && width < 480 ? 0.8 : 1) *
+    //       useSelector(state => state.settings.zoom);
```

Comment:
What made you struggle here? Why not just:
```suggestion
    const originalZoom = useRecoilValue(setZoomState);
    const zoom = isNestedZoom
        ? 1
        : ((lang === 'parallel' || isBibleParallel) && width < 480 ? 0.8 : 1) *
          originalZoom;
```